main.go

```diff
@@ -7,6 +7,7 @@ import (
 	"os"
 
 	"code.cloudfoundry.org/lager"
+	"github.com/BurntSushi/toml"
```

Comment:
Mind running `dep ensure` and checking in the changes?

Reply:
Done.